client/src/components/formComponents/CreatePost.jsx

```diff
@@ -75,18 +81,116 @@ const CreatePost = () => {
       <Typography component="legend" variant="h6">
         What made you smile today, {registeredUser.firstName}?
       </Typography>
+      <form
+        enctype="multipart/form-data"
+        onSubmit={submitHandler}
+        method="POST"
+      >
+        <Paper align="center" variant="outlined" mx="auto" p={1}>
+          <FormControl>
+            {/* Form Starts */}
+            <div className="input-field" row={false} sx={{ p: "5px" }}>
+              {/* NAME INPUT */}
+              <input
+                id="name"
+                value={name}
+                // error
+                onChange={(e) => setName(e.target.value)}
+                type="text"
+              />
+              <label for="name">Title</label>
+              <span class="helper-text" data-error="wrong" data-success="right">
+                {formInputError.name?.message}
+              </span>
+            </div>
 
-      <PostForm
+            {/*  TEXT INPUT */}
+            <div className="input-field" sx={{ p: "5px" }}>
+              <textarea
+                id="text"
+                value={text}
+                maxRows="6"
+                onChange={(e) => setText(e.target.value)}
+                className="materialize-textarea"
+              />
+              <label for="text">What would you like to say?</label>
+              <span class="helper-text" data-error="wrong" data-success="right">
+                {formInputError.text?.message}
+              </span>
+            </div>
+
+            {/* URL INPUT */}
+            <div className="input-field" sx={{ p: "5px" }}>
+              <input
+                id="url"
+                value={url}
+                onChange={(e) => setUrl(e.target.value)}
+                type="url"
+                // input="url"
+              />
+              <label for="url">Add link - optional</label>
+              <span class="helper-text" data-error="wrong" data-success="right">
+                {formInputError.url?.message}
+              </span>
+            </div>
+            {/* IMAGE UPLOAD */}
+            <div className="file-field" sx={{ p: "5px" }}>
+              <div className="btn">
+                <i className="material-icons large prefix">photo_camera</i>
+
+                <input
+                  onChange={(e) => onChangeFileSelectHandler}
```

Comment:
You're not invoking the function unless you add parenthesis
```suggestion
                  onChange={(e) => onChangeFileSelectHandler(e)}
```